events/permissions.py

```diff
@@ -157,7 +158,15 @@ def has_object_permission(self, request: Request, view, obj):
 class DataSourceOrganizationEditPermission(BasePermission):
     def has_permission(self, request, view):
         if request.method == "POST":
-            if (
+            if settings.WEB_STORE_INTEGRATION_ENABLED and request.data.get(
+                "web_store_merchants"
+            ):
+                return (
+                    request.user.is_authenticated
+                    and request.user.is_superuser
+                    and request.user.admin_organizations.exists()
+                )
```

Comment:
Should this be
```
return request.user.is_authenticated and (
                    request.user.is_superuser
                    or request.user.admin_organizations.exists()
                )
```
Now user has to be superuser and admin in the organization.

Reply:
Yes, I think it should check only for "is_superuser" for the merchant changes as that's how the customer/price group management has been implemented (only a superuser or a financial admin can manage them). Here I followed the existing permissions checks a bit too much maybe, since that code requires the user to be an admin of an organization regardless of their superadmin status.

Reply:
Now changed to be like the rest of the financial management permission checks: organization admin membership is not required for a superuser, and the user must be either a superuser or a financial admin. If any other organization data in addition to merchants are being added or updated, the user must be a superuser (as a financial admin cannot modify the organization's other data).

Reply:
Sounds reasonable. In that case user (e.g. `
GET 
http://localhost:8080/v1/user/373d2f73-c8e2-4397-b4d6-fff5be8c935e/
`) endpoint could return `is_superuser` field so the UI would be able to check the permissions as well. At the moment UI doesn't know if current user is superuser or not. That could be added in a separate commit or PR